buildkonfig-gradle-plugin/src/test/kotlin/com/codingfeline/buildkonfig/gradle/BuildKonfigPluginFlavorTest.kt

```diff
@@ -532,4 +532,45 @@ class BuildKonfigPluginFlavorTest {
         Truth.assertThat(iosResult.readText())
             .contains("defaultValue")
     }
+
+    @Test
+    fun `The generated object uses the given objectName`() {
```

Comment:
Could you add a test case for a config below?

```gradle
buildkonfig {
  packageName = "com.example"
  objectName = "AwesomeConfig"

  defaultConfigs {
    buildConfigField 'STRING', 'value', 'defaultValue'
  }

  targetConfigs {
    js {
      buildConfigField 'STRING', 'value', 'jsValue'
    }
  }
}
```

This should generate objects in each target, while the test case you added creates a single object in commonMain.